docs/reference/mapping/dynamic-mapping.asciidoc

```diff
@@ -1,11 +1,11 @@
 [[dynamic-mapping]]
-== Dynamic Mapping
+== Dynamic mapping
 
-One of the most important features of Elasticsearch is that it tries to get
+One of the most important features of {es} is that it tries to get
```

Comment:
Was this on purpose?

Reply:
Yes! The `{es}` variable is rendered as Elasticsearch in the output.

Reply:
For some reason my eyes thought that you removed {es} here. you did the opposite, which is totally fine :)